### PR TITLE
Add Landing Page

### PR DESCRIPTION
**Pull request by pointmekin**
# Notes
- Basically I added the landing page
- The landing page uses the Ticket component, which I updated so that the hole cutout is actually cut out instead of having while circles to blend them with the background (this took much longer than expected lol)
- I updated some eslint rules to use double quotes for consistency (if you like single quotes better, let m know)
- I also updated the auth forms to include it in <form/> component so users can use the enter button to submit forms
- Refactored some code in the landing page
- Refactored messy in-line styles to CSS module instead
- The pipeline still doesn't work due to build errors

### CODE REVIEW

File: components/UI/ticket/Ticket.tsx [hunk 1]
```diff
@@ -58,28 +59,33 @@ function Ticket({
...
       >
         <Box
           sx={{
             backgroundColor: getBgColor(),
-            width: '100%',
-            height: '100%',
-            borderRadius: '12px'
+            width: '100%'
           }}
+          className=""
```
Comment (chuwong35122) [inline components/UI/ticket/Ticket.tsx:88]: unused

File: components/auth/EventOrganizerRegisterForm.tsx [hunk 2]
```diff
@@ -46,7 +46,7 @@ function EventOrganizerRegisterForm() {
   })
 
   return (
-    <>
+    <form onSubmit={handleSubmit}>
```
Comment (chuwong35122) [inline components/auth/EventOrganizerRegisterForm.tsx:49]: should i add <form> to all Forms?

**Comment by chuwong35122** [comment]
Code LGTM

File: components/UI/ticket/Ticket.tsx (hunk 1 above)
Comment (pointmekin) [reply components/UI/ticket/Ticket.tsx:88]: Ok i will remove that

File: components/auth/EventOrganizerRegisterForm.tsx (hunk 2 above)
Comment (pointmekin) [reply components/auth/EventOrganizerRegisterForm.tsx:49]: Sure, but no hurries

**Pull request merged**